packages/api/src/sdk-bundle-api/sdk-to-bundle/network-stubbing.ts

```diff
@@ -53,3 +61,41 @@ export interface ConnectionTypesFilter {
   xhr: boolean;
   webSockets: boolean;
 }
+
+type TransformableRequestData = Pick<Request, "body" | "method" | "url">;
+
+interface CustomTransformationBase {
+  /**
+   * The string to search for in the request component. This can also be a regular expression.
+   */
+  searchString: string;
+  replacement: string;
```

Comment:
```suggestion
  /**
   * The replacement to replace at. This can reference groups in the searchString regex, for example:
   *     - matchRegex = `id_(\w*)=[^&]*`, replacement = `id_$1=<redacted>`
   *     - matchRegex= `id_(?<param_name>\w*)=[^&]*`, replacement = `id_$<param_name>=<redacted>`
   */
  replacement: string;
```